src/main/webapp/login-script.js

```diff
@@ -8,16 +63,16 @@ async function signIn(googleUser) {
   const id = await loginResponse.json();
   if (id.verified) {
     // Successful sign-in.
-    hideLandingElements();
     const termList = await getTermList();
-    const selectElement = document.getElementById("search-term");
-    const optionElement = document.createElement("option");
-    optionElement.appendChild(document.createTextNode("Select term..."));
-    optionElement.value = "";
-    selectElement.appendChild(optionElement);
+    const select = document.getElementById("search-term");
+    const opt = document.createElement("option");
+    opt.appendChild(document.createTextNode("Select term..."));
+    opt.value = "";
+    select.appendChild(opt);
```

Comment:
This is undoing the changes I made to this file - is there a way you can uncommit these changes or pull from master before merging this in?

Reply:
This goes for basically this whole file

Reply:
I can pull from master